docs/models/examples.md

```diff
@@ -61,15 +63,15 @@ x = np.expand_dims(x, axis=0)
 average_pool_8_features = model.predict(x)
 ```
 
-## Fine-tune Bi-Real Net on a new set of classes
+## Fine-tune BinaryDenseNet-45 on a new set of classes
```

Comment:
Have you checked, that the layer numbers in this example make sense with the new model?